great_expectations/data_context/store/inline_store_backend.py

```diff
@@ -208,3 +209,10 @@ def _determine_resource_type_and_name_from_key(
             resource_name = key[1]
 
         return resource_type, resource_name
+
+    def build_key(self, name: str, **kwargs) -> DataContextVariableKey:
```

Comment:
Moved the key generation which is specific to the store backend type to the store backend.

Reply:
love it! should it be a method on the parent? I'd say make it abstract but it looks like we don't really need it for all store backends.